app/src/main/java/com/moxielabs/androidarchitectureexamples/MainActivity.kt

```diff
@@ -1,14 +1,14 @@
-package com.moxielabs.androidarchitectureexamples.shared.views
+package com.moxielabs.androidarchitectureexamples
```

Comment:
Whatcha thinking putting this up one level?

Reply:
It didn't makes sense to me that it was "shared" by anything in the app. mostly because none of the Fragments make reference to it. I didn't stumble upon this until after I had fixed my merge issue. Which was that AS didn't update the Manifest with the location of MainActivity. I'm open to putting it back. Just calling out the implications of calling it "shared"

Reply:
I think, for the sake of this demo, it can live in either place. I agree with you, though; keep it out of `shared`. I see where you're coming from - it's more of a container than a component that's shared across everything in these particular use cases we'll be using it for